MANIFEST.in

```diff
@@ -1 +1,2 @@
-recursive-include larq_compute_engine/ *.so
+recursive-include larq_compute_engine *.so
+recursive-include larq_compute_engine *.pyd
```

Comment:
`pybind_extension` outputs `.pyd` files on Windows so we need to include them in the wheel.